app/graphql/types/profile.rb

```diff
@@ -67,14 +67,19 @@ class Profile < Types::BaseObject
 
     field :major_os_version, String, null: false
 
+    def policy
+      object.old_policy
```

Comment:
Non- blocker: For the UI to leverage the Policy association this should return
```
object.policy_object&.profiles&.external(false)&.first
```
(in short)

Reply:
that should be equivalent to `old_policy`, no?

Reply:
Yes, with some exceptions.